### ISSUE TITLE
[bug] Chunk files missing content hash on production builds

### ISSUE THREAD

**Issue by retlehs** [roots/bud#1766]
### Describe the issue

Chunk files do not contain a hash on a production build

### Expected Behavior

Chunk files should contain a hash, like all other assets do

### Actual Behavior

Chunk files do not contain a hash

### Steps To Reproduce

1. Run a production build that will produce chunk files

### version

6.4.5

**Comment by retlehs** [comment]
Workaround via @kellymears:

```js
bud.hooks.on(`build.output.chunkFilename`, () =>
  bud.hooks.filter(`feature.hash`)
    ? `js/dynamic/[id].[contenthash:6].js`
    : `js/dynamic/[id].js`,
)
```

**Comment by kellymears** [comment]
Should be resolved in [6.5.0](https://bud.js.org/releases/6.5.0/)
